dlp/api/Snippets/README.md

```diff
@@ -18,6 +18,24 @@ A collection of samples that demonstrate how to call the
     PS > $env:GOOGLE_PROJECT_ID = 'YOUR-GOOGLE-PROJECT-ID'
     ```
 
+6. Set the environment variable `GCS_FILE_URI` to your Google Cloud Storage file path.
+    
+    ```ps1
+    PS > $env:GCS_FILE_URI = 'YOUR-GCS-FILE-URI'
+    ```
+
+7. Set the environment variable `TOPIC_ID` to your Pub/Sub Topic ID.
+
+    ```ps1
+    PS > $env:TOPIC_ID = 'YOUR-TOPIC-ID'
+    ```
+
+8. Set the environment variable `SUBSCRIPTION_ID` to your Pub/Sub Subscription ID.
```

Comment:
Please call out that this is a subscription to the topic above.

Reply:
Done